SConstruct

```diff
@@ -339,6 +342,56 @@ if selected_platform in platform_list:
 	if (env_base['squish']=='yes'):
 		env.Append(CPPFLAGS=['-DSQUISH_ENABLED']);
 
+	if (env['webm']=='yes'):
+		env['vorbis']='yes'
+		env['opus']='yes'
+		env.Append(CPPFLAGS=['-DWEBM_ENABLED']);
+
+		env['webm_cpu_arm'] = selected_platform == 'winrt' or selected_platform == 'iphone' or selected_platform == 'bb10' or (selected_platform == 'android' and env["android_arch"] != 'x86')
+		env['webm_cpu_x86'] = (env["bits"] == "32" or env["bits"] == "64") and (selected_platform == 'windows' or selected_platform == 'x11' or selected_platform == 'osx' or selected_platform == 'haiku' or (selected_platform == 'android' and env["android_arch"] == 'x86'))
+
+		if env['webm_cpu_x86'] and Execute("yasm --version"):
```

Comment:
This `Execute` fails on Windows even with YASM in the PATH. Replacing it with `os.system` works.